package.json

```diff
@@ -9,8 +9,10 @@
     "url": "https://github.com/jenkins-x/jx-docs.git"
   },
   "dependencies": {
+    "docsearch.js": "^2.6.3"
```

Comment:
why is this dependency moved from _devDependency_ to _dependency_? Is this intentional?